### DISCUSSION TITLE
ESP32 D1 mini and a VL53L0X laser ranging module

### DISCUSSION THREAD

**Discussion by jnicke** [arendst/Tasmota#11646] [Q&A] [answered]
Hello,

I have an ESP32 D1 mini and a VL53L0X laser ranging module here. Is this even supported by tasmota32?
It is listed in the overview of the Supported I2C devices. Unfortunately it is not recognized by the ESP32. I2CScan returns "device found at 0x29".
"I2CDriver": "! 7,8,9,10,11,12,13,14,15,! 16,17,18,20,24,29,36,41,42,44,46,48". The 31 for the VL53L0X is missing.
Is there a way to get the sensor to work on an ESP32?
I installed the current tasmota32-sensors.bin

**Comment by sfromis** [comment, marked as answer]
Take it easy with cross-posting.... I just answered you on Discord:

As https://github.com/arendst/Tasmota/blob/development/BUILDS.md shows, many sensors including VL53L0X are not included in any precompiled binaries, you need to https://tasmota.github.io/docs/Compile-your-build/
BTW: tasmota32-sensors.bin is being phased out, in dev it has been merged into tasmota32.bin
